alembic/env.py

```diff
@@ -18,7 +18,7 @@
 # access to the values within the .ini file in use.
 config = context.config
 
-config.set_main_option('sqlalchemy.url', os.getenv('SQL_ALCHEMY_URL'))
+config.set_main_option('sqlalchemy.url', os.getenv('ALEMBIC_DB_URL'))
```

Comment:
It is not a big deal, but why did you rename this env variable to 'ALEMBIC_...'? If it is a connection string to db it is not related to alembic. Even if we were not using Alembic, the same value would be there, so some names related to db might be a better option.